cmd/airflow.go

```diff
@@ -370,7 +388,18 @@ func airflowRun(cmd *cobra.Command, args []string) error {
 
 	// Add airflow command, to simplify astro cli usage
 	args = append([]string{"airflow"}, args...)
-	return airflow.Run(config.WorkingPath, args)
+	// ignore last user parameter
+	return airflow.Run(config.WorkingPath, args, "")
+}
+
+// airflowUpgradeCheck
+func airflowUpgradeCheck(cmd *cobra.Command, args []string) error {
+	// Silence Usage as we have now validated command input
+	cmd.SilenceUsage = true
+
+	// Add airflow command, to simplify astro cli usage
+	args = append([]string{"bash", "-c", "pip install --user astro --no-deps 'apache-airflow-upgrade-check'; airflow upgrade_check"})
```

Comment:
```suggestion
	args = append([]string{"bash", "-c", "pip install --user --no-deps 'apache-airflow-upgrade-check'; airflow upgrade_check"})
```

The `--user` option doesn't take an argument, it is simply a boolean flag

Reply:
This is installing https://pypi.org/project/astro/ :grin:

Reply:
Since we are running it as root (L402), I don't think we need `--user` flag too